dockyard/api/config.py

```diff
@@ -0,0 +1,24 @@
+# Pecan Application Configurations
+app = {
+    'root': 'dockyard.controllers.root.RootController',
+    'modules': ['dockyard', 'dockyard.api'],
+    'static_root': '%(confdir)s/public',
+    'template_path': '%(confdir)s/dockyard/templates',
```

Comment:
As per documentation: 
Points to the directory where your template files live (relative to the project root).

Discuss and make sure that it should be removed if not required in this project.